packages/amplify-data-construct/API.md

```diff
@@ -0,0 +1,20 @@
+## API Report File for "@aws-amplify/data-construct"
```

Comment:
Aren't there other types that should be exported from the this package as well?

Reply:
Yeah, working through that now. I probably won't have a good way to guarantee they're in sync, but it should be pretty do-able.

Reply:
Added!